src/um/pdh.rs

```diff
@@ -54,3 +222,338 @@ STRUCT!{struct PDH_COUNTER_PATH_ELEMENTS_W {
+    hCounter: PDH_HCOUNTER, dwFormat: DWORD, rawValue1: PPDH_RAW_COUNTER,
+    rawValue2: PPDH_RAW_COUNTER, fmtValue: PPDH_FMT_COUNTERVALUE
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhComputeCounterStatistics(
+    hCounter: PDH_HCOUNTER, dwFormat: DWORD, dwFirstEntry: DWORD, dwNumEntries: DWORD,
+    lpRawValueArray: PPDH_RAW_COUNTER, data: PPDH_STATISTICS
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhGetCounterInfoW(
+    hCounter: PDH_HCOUNTER, bRetrieveExplainText: BOOLEAN, pdwBufferSize: LPDWORD,
+    lpBuffer: PPDH_COUNTER_INFO_W
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhGetCounterInfoA(
+    hCounter: PDH_HCOUNTER, bRetrieveExplainText: BOOLEAN, pdwBufferSize: LPDWORD,
+    lpBuffer: PPDH_COUNTER_INFO_A
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhSetCounterScaleFactor(
+    hCounter: PDH_HCOUNTER, lFactor: LONG
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhConnectMachineW(
+    szMachineName: LPCWSTR
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhConnectMachineA(
+    szMachineName: LPCSTR
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumMachinesW(
+    szDataSource: LPCWSTR, mszMachineNameList: LPWSTR, pcchBufferLength: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumMachinesA(
+    szDataSource: LPCSTR, mszMachineNameList: LPSTR, pcchBufferLength: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumObjectsW(
+    szDataSource: LPCWSTR, szMachineName: LPCWSTR, mszObjectList: LPWSTR,
+    pcchBufferLength: LPDWORD, dwDetailLevel: DWORD, bRefresh: BOOL
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumObjectsA(
+    szDataSource: LPCSTR, szMachineName: LPCSTR, mszObjectList: LPSTR,
+    pcchBufferLength: LPDWORD, dwDetailLevel: DWORD, bRefresh: BOOL
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumObjectItemsW(
+    szDataSource: LPCWSTR, szMachineName: LPCWSTR, szObjectName: LPCWSTR,
+    mszCounterList: LPWSTR, pcchCounterListLength: LPDWORD, mszInstanceList: LPWSTR,
+    pcchInstanceListLength: LPDWORD, dwDetailLevel: DWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumObjectItemsA(
+    szDataSource: LPCSTR, szMachineName: LPCSTR, szObjectName: LPCSTR,
+    mszCounterList: LPSTR, pcchCounterListLength: LPDWORD, mszInstanceList: LPSTR,
+    pcchInstanceListLength: LPDWORD, dwDetailLevel: DWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhMakeCounterPathW(
+    pCounterPathElements: PPDH_COUNTER_PATH_ELEMENTS_W, szFullPathBuffer: LPWSTR,
+    pcchBufferSize: LPDWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhMakeCounterPathA(
+    pCounterPathElements: PPDH_COUNTER_PATH_ELEMENTS_A, szFullPathBuffer: LPSTR,
+    pcchBufferSize: LPDWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhParseCounterPathW(
+    pCounterPathElements: *mut PDH_COUNTER_PATH_ELEMENTS_W, szFullPathBuffer: LPWSTR,
+    pcchBufferSize: LPDWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhParseCounterPathA(
+    pCounterPathElements: *mut PDH_COUNTER_PATH_ELEMENTS_A, szFullPathBuffer: LPSTR,
```

Comment:
The first two arguments to `PdhParseCounterPathA` and `PdhParseCounterPathW` [seem to be switched](https://msdn.microsoft.com/en-us/library/windows/desktop/aa372657(v=vs.85).aspx)